### ISSUE TITLE
No (valid) module for script 'load_data' found

### ISSUE THREAD

**Issue by codernirdesh** [AtitBimali/django-nepal#1]
Hi, I'm very new to Python and Django but I was implementing an address feature in an app.

While running the script i got this error, tried to solve it using chatGPT and Google, StackOverflow but didn't find anything solving.

I am getting following things:

Code ran : 
```bash
$ python manage.py runscript load_data
```

Result : 
```console
  No (valid) module for script 'load_data' found
  Try running with a higher verbosity level like: -v2 or -v3
  CommandError: An error has occurred running scripts. See errors above.
```

**Comment by AtitBimali** [comment]
@codernirdesh thanks for raising the issue, this error has been here for a quite a while. I have a quick fix to the problem but it's not ideal. I will release a newer version soon.

**Comment by codernirdesh** [comment]
I think the error is there because you `.gitignore`d the csv data file.

I looked into the code and found that in a recent push, you gitIgnored the file.

Please have a look at it. @AtitBimali

**Comment by AtitBimali** [comment]
@codernirdesh, thanks for pointing it out! I've got a handle on that missing CSV file issue you flagged. However, before it even gets to that point, there's another hurdle—looks like the scripts folder is MIA.

No worries, though. I'll have a fix ready to roll shortly.

**Comment by AtitBimali** [comment]
The reported issue has been successfully addressed and resolved in the latest release, [version 0.0.9](https://github.com/AtitBimali/django-nepal/releases/tag/v0.0.9) 
If you encounter any further issues or have additional feedback, please don't hesitate to reopen or create a new issue.
